### PR TITLE
CASMPET-5117: Refactor keycloak-plugin scripts

### PR DESCRIPTION
**Pull request by zcrisler**
The keycloak-plugin scripts have been refactored and redistributed to other locations:
- `configure-jvm-cacerts` has been refactored as the more generic `create-trust-store` utilty
- `generate-credential-config` has been replaced with a Kubernetes Secret in the cray-nexus chart
- `update-realm` has been replaced with `nexus-enable-keycloak-realm` and `nexus-disable-keycloak-realm` scripts in the cray-nexus-setup container image
- `initialize-keycloak-plugin` has been replaced by integrating `create-trust-store` into the Nexus initContainer and adding `nexus-*-keycloak-realm` scripts to the post-install and post-upgrade Helm hook in the cray-nexus chart

### CODE REVIEW

File: create-trust-store.sh [hunk 1]
```diff
@@ -0,0 +1,31 @@
+#!/bin/sh
+
+# Copyright 2022 Hewlett Packard Enterprise Development LP
+
```
Comment (rnoska-hpe) [inline create-trust-store.sh:4]: Is this running from within the context of the Nexus pod shell?  If so I don't think these messages will appear in the pod log.

I think you will need to add something similar to:
`log() {
  LOG=/proc/1/fd/1
  if [[ ! -z "$1" ]]; then echo "$(date) [$0] $1" | tee $LOG; fi
}`

Comment (zcrisler) [reply create-trust-store.sh:4]: This runs from the initContainer, not the nexus pod.

**Comment by zcrisler** [comment]
Successfully tested a pre-release tag of the updated image as part of a Nexus upgrade on wasp.

**Review by rnoska-hpe — APPROVED** [review]
Approving based on initial testing on Wasp.

**Pull request merged**